### PR TITLE
community/hugo: upgrade to 0.39

### PR DESCRIPTION
**Pull request by tboerger**
Updated to latest version, requires the `dep` package within `community` which have been moved my @clandmeter.

### CODE REVIEW

**Comment by tboerger** [comment]
I have dropped all `return 1` statements.

**Comment by tboerger** [comment]
After removing `|| return 1` the Travis build fails :(

File: community/hugo/APKBUILD [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+	dep ensure -v
+	go build -v -o bin/$pkgname
 }
 
 check() {
 	rm -f $builddir/releaser/git_test.go
 	cd "$builddir"
 	export GOPATH="$srcdir"
-	govendor test +local || return 1
+	go test
```
Comment (clandmeter) [inline community/hugo/APKBUILD]: >>> hugo*: check
?       github.com/gohugoio/hugo        [no test files]

Could this be the issue? can you check what going on here?

Comment (tboerger) [reply community/hugo/APKBUILD]: Ah yeah, it doesn't find tests in the root package. But this exits with 0.

Comment (tboerger) [reply community/hugo/APKBUILD]: I have enabled the tests recursively now.

**Comment by tboerger** [comment]
Tests are patched to work now, but Travis still fails for a different reason.

**Comment by andypost** [comment]
@tboerger please rebase commit on top of current master to fix travis build

**Pull request closed without merging**